src/store/wizardSlice.ts

```diff
@@ -61,6 +73,12 @@ const initialState: wizardState = {
     registrationType: 'register-now-rhc',
     activationKey: '',
   },
+  profile: undefined,
+  openScap: {
+    profileId: undefined,
+    profileName: undefined,
+    profileDescription: undefined,
+  },
```

Comment:
This would need to get updated as per the comment above. Having an `openScap` as a step, and `profile` and `profileInformation` as a information it contains.